data/team/members.yaml

```diff
@@ -104,18 +75,25 @@
   key: andrea-lilian.morger
   role: PhD candidate
   status: member
-  image: /images/team/andreamorger.jpg
-  project: In silico toxicity prediction and application to external data
+  project: Machine learning and method development for kinase drug discovery
+  image: /images/team/coreytaylor.jpg
   links:
     - icon: envelope
-      text: andrea-lilian.morger@charite.de
-      url: mailto:andrea-lilian.morger@charite.de
+      text: corey.taylor@charite.de
+      url: mailto:corey.taylor@charite.de
+    - icon: linkedin
+      text: LinkedIn
+      url: https://www.linkedin.com/in/corey-taylor-264050208/
     - icon: github
       text: GitHub
-      url: https://github.com/morgeral
+      url: https://github.com/corey-taylor
+    - icon: graduation-cap
+      text: Google Scholar
+      url: https://scholar.google.com/citations?user=IjR7hksAAAAJ&hl=en
     - icon: book
       text: ORCiD
-      url: https://orcid.org/0000-0003-4774-6291
+      url: https://orcid.org/0000-0003-2535-9514
```

Comment:
This bit still renders incorrectly locally (`hugo serve`). Andrea M. has your information right now.

Reply:
@AndreaVolkamer do you want her as an alumni already? If so, this bit can be deleted entirely.

Reply:
> @AndreaVolkamer do you want her as an alumni already? If so, this bit can be deleted entirely.

Yes I just did that. There was duplication.

Reply:
Ok, resolved now!

Reply:
@corey-taylor lets keep Andrea M. in for now, until she submitted her thesis